dev/devicelab/lib/framework/framework.dart

```diff
@@ -147,10 +118,40 @@ class _TaskRunner {
     } finally {
       print('Cleaning up after task...');
       await forceQuitRunningProcesses();
+      await checkForRebootRequired();
       _closeKeepAlivePort();
     }
   }
 
+  Future<void> checkForRebootRequired() async {
+    try {
+      final Device device = await devices.workingDevice.timeout(const Duration(seconds: 15));
+      if (noRebootForbidList.contains(device.deviceId)) {
+        return;
+      }
+      final File rebootFile = File('.reboot-count');
```

Comment:
what path is this file based on? If this is based on cocoon or flutter repo, it may be affected when refreshing new versions. Shall we put this file somewhere like `/home/flutter`?